follow_fork/README.md

```diff
@@ -0,0 +1,27 @@
+Follow fork()
+=============
+
+Often when recording engineers might want to record both the parent and the
+children generated during the run.
+This pre-load library intercepts the calls to fork() and calls the
+[LiveRecorder API](http://undo.io) to record the child.
+
+Note
+----
+
+You will need to have the LiveRecorder library in order to be able to
+use this utility.
+
+Compiling
+---------
+
+In order to be able to use the library it needs to be compiled with
+the following command:
+
+Markup: `gcc -I /<path>/<to>/<undodb-xxxx>/undolr -L /<path>/<to>/<undodb-xxxx>/undolr -shared -fPIC follow_fork.c -o libfollowfork.so -l:libundolr_pic_x64.a -std=c99 -ldl`
```

Comment:
What does `Markup: ` mean? 🤔

You could use triple-backticks here and for the next example.

Reply:
I copied the rules from https://github.com/tchapi/markdown-cheatsheet/blob/master/README.md
but I realize now that I think I've been stupid: Markdown shows what the markdown is, it is not meant as a keyword... removing